src/Nancy/Json/Simple/NancySerializationStrategy.cs

```diff
@@ -72,8 +72,9 @@ protected override string MapClrMemberNameToJsonFieldName(string clrPropertyName
         /// </summary>
         /// <param name="value">The object to deserialize</param>
         /// <param name="type">The type of object to deserialize</param>
+        /// <param name="dateTimeStyles">The <see cref="DateTimeStyles"/> ton convert <see cref="DateTime"/> objects</param>
         /// <returns>A instance of <paramref name="type" /> deserialized from <paramref name="value"/></returns>
-        public override object DeserializeObject(object value, Type type)
+        public override object DeserializeObject(object value, Type type, DateTimeStyles dateTimeStyles)

```

Comment:
This should probably be passed into the ctor, just like `bool retainCasing` and have the default ctor fallback to `JsonConfiguration.Default.DateTimeStyles`